### PR TITLE
fix: add an error status message when vote fails

### PR DESCRIPTION
**Pull request by xichi**
closes https://github.com/DimensionDev/Maskbook/issues/2137

### CODE REVIEW

File: packages/maskbook/src/_locales/zh/messages.json [hunk 1]
```diff
@@ -392,5 +392,6 @@
     "plugin_poll_status_closed": "投票已截止",
     "plugin_poll_status_voting": "投票中",
     "plugin_poll_status_voted": "已投票.",
+    "plugin_poll_status_error": "蔙誤!",
```
Comment (septs) [inline packages/maskbook/src/_locales/zh/messages.json:395]:
```suggestion
    "plugin_poll_status_error": "錯誤!",
```

File: packages/maskbook/src/plugins/Polls/UI/Polls.tsx [hunk 2]
```diff
@@ -126,9 +135,12 @@ export function PollCardUI(props: PollCardProps) {
                     <ListItem
                         className={classes.option}
                         key={index}
-                        onClick={() => {
-                            vote?.(poll, index)
-                        }}>
+                        onClick={debounce(
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/Polls/UI/Polls.tsx:138]: IMO it will create a new "debounced" button every time so it will not work

Comment (Tedko) [reply packages/maskbook/src/plugins/Polls/UI/Polls.tsx:138]: @1999-XiChi cc

**Comment by septs** [comment]
outdated

**Pull request closed without merging**